requirements-integration.in

```diff
@@ -1,9 +1,9 @@
 aiohttp
 jinja2
 juju<3.1
+lightkube
```

Comment:
I'm not sure if I missed this from the previous version of this PR, but why are we adding it and removing the requirements.txt file ?

Reply:
Yeah that's fair.